src/bundle/Security/Http/Authenticator/Passport/TwoFactorPassport.php

```diff
@@ -39,4 +42,13 @@ public function getFirewallName(): string
     {
         return $this->twoFactorToken->getProviderKey();
     }
+
+    public function getUser(): UserInterface
+    {
+        $user = $this->twoFactorToken->getUser();
+        if($user instanceof UserInterface) {
```

Comment:
Would you please fix the code style? Have a look what the code style build is complaining about. Running `php-cs-fixer fix` might actually do the job.